cli/src/cli_builder.rs

```diff
@@ -227,7 +227,7 @@ impl<'a> NewCli<'a> {
             cacert,
             insecure,
             timeout,
-        } = OxideCli::from_arg_matches(&matches).unwrap();
+        } = OxideCli::from_arg_matches(&matches)?;
```

Comment:
Is this testable? If not let's leave it an unwrap or replace with expect or whatever

Reply:
👍